src/styles/blocks/star.scss

```diff
@@ -0,0 +1,16 @@
+.stars {
+  display: flex;
+
+  &__star {
+    height: 16px;
+    width: 16px;
+    background-image: url('../images/star.svg');
+    background-repeat: no-repeat;
+    background-position: center;
+    margin-right: 4px;
+  }
+}
+
+.stars--rating-4 .stars__star:nth-child(-n + 4) {
+  background-image: url('../images/star-active.svg');
+}
```

Comment:
It would be a good idea to provide styles for all possible star ratings (from 1 to 5). This way, you can easily change the rating by just modifying the class of the parent element. For example, you can add styles for `.stars--rating-1`, `.stars--rating-2`, etc., each with the appropriate number of active stars.

Reply:
> It would be a good idea to provide styles for all possible star ratings (from 1 to 5). This way, you can easily change the rating by just modifying the class of the parent element. For example, you can add styles for `.stars--rating-1`, `.stars--rating-2`, etc., each with the appropriate number of active stars.

Good point